src/js/components/Chart.vue

```diff
@@ -0,0 +1,341 @@
+<template>
+  <div class="chart-container">
+    <span class="revenue-label">Total <strong>Revenue</strong></span>
+    <svg class="revenue-line" width="100%" height="310px">
+      <line :y1="Y0" :y2="Y0 + HEIGHT * scale" x1="0" x2="0" />
+    </svg>
+    <svg id="chart" data-chart width="100%" height="310px">
```

Comment:
I think we can make this responsive by giving it a viewbox, and moving height/width into CSS?